### PR TITLE
adding foreman-debug info

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

File: support.md [hunk 1]
```diff
@@ -5,6 +5,23 @@ title: Support
 
 There are two primary methods of getting support for the Foreman: IRC and mailing lists.
 
+### Gathering information
+
+To get appropriate support, you need to provide sufficient information about
+operating system, environment and setup. You can utilize foreman-debug tool
+which collects information about OS, Foreman and related components. If you
+installed from packages, it is available as command for root:
```
Comment (domcleal) [inline support.md]: "about your operating system [..]"

"You can utilize the foreman-debug [..]"

"information about your OS, Foreman [..]"

"[..] from packages, the command is available to root:"

**Comment by domcleal** [comment]
:+1:, though I will wait until 1.2 is in RC stage and we have foreman-debug packaged before merging.  I've noted a few readability improvements above too.

**Comment by lzap** [comment]
Okay, pushed.

LZ

On Tue, May 07, 2013 at 01:38:45AM -0700, Dominic Cleal wrote:

> :+1:, though I will wait until 1.2 is in RC stage and we have foreman-debug packaged before merging.  I've noted a few readability improvements above too.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/theforeman/theforeman.org/pull/46#issuecomment-17530165

## 

Later,

 Lukas "lzap" Zapletal
 irc: lzap #theforeman

File: support.md [hunk 2]
```diff
@@ -5,6 +5,24 @@ title: Support
...
 
+### Gathering information
+
+To get appropriate support, you need to provide sufficient information about
+your operating system, environment and setup. You can utilize the
+foreman-debug tool which collects information about your OS, Foreman and
+related components. If you installed from packages, the command is available
+to root:
+
+    # foreman-debug
```
Comment (domcleal) [inline support.md:16]: We didn't do this in the packaging ([#2558](http://projects.theforeman.org/issues/2558)), so if we can fix that or change this, I'll merge this now 1.2 RCs are out.

**Pull request merged**